docs/Dialects/LTL.md

```diff
@@ -176,14 +201,22 @@ The `ltl.delay` sequence operation represents various shorthands for the *next*/
 | `ltl.delay %a, 2`    | **XXF**a                    |
 
 
-### Concatenation
+### Until, Always and Eventually
+
+`ltl.until` is *weak*, meaning the property will hold even if the trace does not contain enough clock cycles to evaluate the property. `ltl.eventually` is *strong*, where `ltl.eventually %p` means `p` must hold at some point in the trace.
+
+`always` is not implemented for now, but `always p` could be expressed by `not s_eventually not p`. Notice that assertions work globally. Properties will be checked from every cycle.
```

Comment:
Here I mean the operator is not implemented. We can't write something like `p |-> always q` for now.
As for `not s_eventually not p`, maybe we can just leave it here until we have the plan to implement or encode `always`.

Reply:
It's a question of semantic meaning, i.e. `p |-> always q` and `always p |-> q` are semantically equivalent, but  `p |-> eventually q` and `eventually p |-> q` are not. This is one of the annoying subtleties of the always so in practice leaving it as the default will make it easier to encode. I'm just afraid that telling people to encode it as the negated liveness of a negated property will make it impossible to encode as anything other than a vendor-specific implementation of SVA. The end goal here is to have something that we can implement in open-source simulation, rather than relying on things that require weird heuristics to implement. This is just my opinion, but I think this particular sentence will lead to a lot of trouble in the future.

Reply:
It's a mistake I use `p |->` because `|->` in SVA only allows a sequence on the left side. However, we can consider the `|->` as the logical implacation in this conversation.
`p |-> always q` means if `p` is true, then `q` needs to be always true for the reset of the time. But `(always p) |-> q` means if `p` is always true, then `q` needs be true. Or `always (p |-> q)` means every time `p` is true, then `q` needs to be true. They don't seem equivalent. 

It's precisely because simulation that I don't think we need to worry about this liveness. In simulation, it dosen't actually check if the property eventually holds on the infinite trace, but only evaluates the property based on the existing simulation trace. Please let me know if I have a misunderstanding about what we want the simulation do with the properties.